deploy/examples/monitoring/prometheus-service.yaml

```diff
@@ -6,10 +6,10 @@ metadata:
 spec:
   type: NodePort
   ports:
-  - name: web
-    nodePort: 30900
-    port: 9090
-    protocol: TCP
-    targetPort: web
+    - name: web
```

Comment:
How about revert the prometheus whitespace changes in the monitoring folder? Not sure why these were updated?

Reply:
ah makes sense if it's for the linter. Why weren't they caught before? I thought you were just refactoring the makefil rules.

Reply:
@travisn:  Indeed, this is about refactoring functionality out of CI workflows into Makefile targets. But locally running `make yamllint` revealed those errors. I started to fix them as part of this patchset since I did not want to push something into the PR of which I already knew it would fail CI. I have no clue why these errors have not been caught by ci before. Possibly slight differences in the version of yamllint used. Making sense?

If requested, it will be easy to revert the yaml fixes as they are in a separate commit.

Reply:
Similar to the yaml errors caused in the shellcheck action, these prometheus changes look like they are are also having indentation issues that will cause problems. At least with the github actions we can immediately see if the formatting is ok, but I worry that these others will be broken. How did you make the updates? Individual lines based on the errors? Instead, let's find a tool that will reformat the whole file, so it should be more reliable.

Reply:
I'd prefer not to make this change here, or any changes to yaml files. Changing yaml files at the same time as the make/CI targets involving yamllint is still implying to me that something has changed in the way yamllint is running, which is not the intent of the PR.

Reply:
I'm wondering if the linter has not been reporting yaml linter failures in the CI, and the check is fixed now with Michael's refactoring. I tried the yamllinter locally and confirmed that these yaml fixes are the minimal needed for the linter to pass.